db/schema.rb

```diff
@@ -292,6 +292,7 @@
     t.datetime "updated_at"
     t.float "max_score", default: 0.0
     t.boolean "optional", default: false
+    t.boolean "starred", default: false
```

Comment:
maybe git pull from main branch to get rid of the schema addition since it's not tied to this specific pr?

Reply:
+1, there should be no changes to `schema.rb` in this PR